cfme/tests/containers/test_ssl_providers.py

```diff
@@ -72,7 +72,9 @@ def test_add_provider_naming_conventions(provider, appliance, soft_assert):
                 'Containers Providers "' + provider_name + '" was saved')
         except AssertionError:
             soft_assert(False, provider_name + ' wasn\'t added successfully')
-        ContainersProvider.clear_providers()
```

Comment:
Removed clear_providers() in favor of just deleting the provider that was added by the test.

Reply:
**QUESTION** I think this is OK, but I wonder if we should add a wait to the clear_providers too......